src/foam/nanos/u2/navigation/UserInfoNavigationView.js

```diff
@@ -16,9 +16,8 @@ foam.CLASS({
   ],
 
   imports: [
-    'agent',
-    'user',
-    'theme'
+    'theme',
```

Comment:
Is this used?

Reply:
yes, line67
this.Menu.create({ id: this.theme.settingsRootMenu })